### ISSUE TITLE
Add an option to disable i18n support

### ISSUE THREAD

**Issue by Cryrivers** [Turbo87/intellij-emberjs#114]
`defaultLocale` is not always a constant in project. If it is a variable, intellij-embers would fail to read translation and result into the screenshot below.

![screen shot 2016-12-16 at 4 15 20 pm](https://cloud.githubusercontent.com/assets/1313238/21256084/13b9d5c4-c3ab-11e6-8d90-2faf21edb10d.png)

So I think we should have an option to disable i18n support.

**Comment by Turbo87** [comment]
indeed, we should probably just disable support for this feature if the locale can't be read
